include/z64save.h

```diff
@@ -7,13 +7,13 @@
 typedef struct {
     /* 0x00 */ u8 buttonItems[4];
     /* 0x04 */ u8 cButtonSlots[3];
-    /* 0x08 */ u16 equipment;
+    /* 0x08 */ u16 equipment; // a mask where each half-byte corresponds to a type of equipment `EquipmentType`, and each half-byte is a piece `EquipValue*`
```

Comment:
I didn't know there was a word for that. Yeah I'll use nibble, even if it's uncommon (?) at least it's clearly defined

Reply:
I'd prefer nybble since byte has a 'y', but it should be well-known, at least.

Reply:
The main spelling seems to be nibble according to wikipedia so I'll go with that since the goal would be using common terms 😄 (also wikipedia does state "half-byte" as an alternative so I too was only half-wrong 👀)